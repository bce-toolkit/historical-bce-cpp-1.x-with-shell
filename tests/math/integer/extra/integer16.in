5936859276775405999043463163850988566159115951665466018901396
4682043595863036147218367755746234735573081778981795149208731230916039038289527633494034750811276780875756986424934718272964
