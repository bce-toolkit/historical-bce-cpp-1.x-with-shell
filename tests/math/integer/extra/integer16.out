4682043595863036147218367755746234735573081778981795149208731236852898315064933632537497914662265347034872938090400737174360
-4682043595863036147218367755746234735573081778981795149208731224979179761514121634450571586960288214716641034759468699371568
27796633956366346068151894447152772344833655271264612707697115345776600581631403147615862427827559969754101702782439398435781242704877701688185838219283408406708951863315645449728657744
0
5936859276775405999043463163850988566159115951665466018901396
0
1
1
1
0
0
